app/Http/Controllers/UsersController.php

```diff
@@ -45,10 +47,41 @@ public function index()
     public function store(Request $request)
     {
         $this->validate($request, $this->registrar->rules());
-        $this->registrar->create($request->all());
+        $user = $this->registrar->create($request->all());
 
-        // @TODO: Add user to transactional bucket through Message Broker!
+        // Now that we've registered a user, add them to the transactional bucket.
+        $payload = [
+            // User information
+            'first_name' => $user->first_name,
+            'birthdate_timestamp' => strtotime($user->birthdate), // Message Broker expects UNIX timestamp
+            'country_code' => $user->country_code,
+        ];
 
+        // Send fields for domestic users
+        if($user->country_code === 'US') {
+            $payload['mobile'] = $user->phone;
+            $payload['mobile_tags'] = [
+                env('APP_NAME_TAG', 'votingapp'),
+            ];
+        }
```

Comment:
I feel like it makes each block a bit more readable to treat them as separate "paragraphs", but it'd work just as well with an `} else {`.